translations/Indonesian/general-rules.md

```diff
@@ -0,0 +1,35 @@
+# General Rules
+
+## Should translations be formal or informal?
+Translations should be formal. One of the reason is because other social networks use formal style, informal translations may not sound friendly.
+**Example:** The word "Delete" can be translated into "Hapus" (formal) or "Buang" (informal). But "Buang" sounds a bit rude, I prefer using "Hapus".
+
+## Which grammatical person should be used for translations (e.g. second person singular, first person plural)?
+Use grammatical person as appropriate to the language. First person singular and second person singular is mostly used, the rest are rarely used.
+
+## What verbs should be used (e.g. active, imperative, continuous)?
+It depends, but translations mostly use imperative and active verbs.
+**Active verb example:** "Ubah" (Change).
+**Imperative verb example:** "Berikan" (Give).
+**Continuous verb example:** "Memuat" (Loading).
+Indonesian continuous verbs usually starts with "men-".
+
+## What types words or phrases should not be used (e.g. slang, regional)?
+Slang or regional words and phrases should not be used.
+Regional word example: "Naon" (Sundanese) = "Apa" (Indonesian) = "What" (English)
+Slang word example: "Doang" (Indonesian slang) = "Hanya" (Indonesian) = "Only" (English)
+Why shouldn't they be used? 
+Regional words: Because it may confuse some people, Over 300 different native languages are spoken in Indonesia, and not everyone can speak fluently in 300 languages.
+Slang words: Because it may sound too rude and may be too informal.
+
+## Consistent usage of Zulip-specific terms and common verbs for actions
+This is important. Most websites that are translated to Indonesian are always formal. (e.g. "beranda", "utama")

```

Comment:
Could you please maybe give English translations of `beranda` and `utama` here? :)

Reply:
Alright!